components/ws-manager-bridge/src/workspace-instance-controller.ts

```diff
@@ -81,17 +81,17 @@ export class WorkspaceInstanceControllerImpl implements WorkspaceInstanceControl
                 const span = TraceContext.startSpan("controlInstances");
                 const ctx = { span };
                 try {
-                    log.debug("Controlling instances...", { installation });
+                    log.debug("Controlling instances...", { installation: workspaceClusterName });
```

Comment:
why do you keep referring to `installation` in the logs?

Reply:
Just fixed that. Was a left-over rename